app/assets/javascripts/Components/tag_table.jsx

```diff
@@ -18,7 +18,7 @@ class TagTable extends React.Component {
 
   fetchData = () => {
     $.get({
-      url: Routes.assignment_tags_path(this.props.assignment_id),
+      url: Routes.course_tags_path(this.props.course_id),
```

Comment:
We still need to pass the assignment_id to the tags controller when this is called from this component:

```ruby
Routes.course_tags_path(this.props.course_id, assignment_id: this.props.assignment_id)
```

We don't have a view for all tags yet